packages/binding-builder-cli/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@rspack/binding-builder-cli",
+  "version": "1.4.2",
```

Comment:
I don't fully understand why we need make our own cli instead of using napi-cli? can you help explain in README.md so outside contributors can understand it

Reply:
Being explained in https://github.com/web-infra-dev/rspack/pull/10876#issuecomment-3027471751